jamesg31/Dispatch-Panel
# v0.3.2

# Implement postal code display for the "transmitLog" component
## Summary of Changes:
Add a postal code display next to the locations to assist with situational awareness. The default postal code that is set for the project is from [DevBlocky](https://github.com/DevBlocky/nearest-postal/blob/master/new-postals.json)'s new-postals.json file which is used by most communities on FiveM.

## Testing Instructions:
- Launch the panel and log into a server
- Go to a location and take a note of the postal in game
- Look on the panel and key up.
   - You should be able to see your postal code next to your current location in brackets
